packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -121,13 +123,7 @@ module.exports = {
       // "url" loader embeds assets smaller than specified size as data URLs to avoid requests.
       // Otherwise, it acts like the "file" loader.
       {
-        exclude: [
-          /\.html$/,
-          /\.(js|jsx)$/,
-          /\.css$/,
-          /\.json$/,
-          /\.svg$/
-        ],
+        exclude: [/\.html$/, /\.(js|jsx)$/, /\.css$/, /\.json$/, /\.svg$/],
```

Comment:
It would get split if they go over the limit (80 characters). I honestly think it's easier to get used to it than ever worry about style nitpicks.